### PR TITLE
Feature/quantity panel

### PR DESCRIPTION
**Pull request by gmverdon**
Quantity panel and sell functionality

### CODE REVIEW

**Comment by gmverdon** [comment]
To be able to set the quantity of your trades.

**Comment by rafaelklaessen** [comment]
No I meant the extra newline

File: src/components/AlertMessage/index.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+const AlertMessage = ({ alert, onDismiss }) => (
+  <Alert color={alert.color} isOpen={alert.isOpen} toggle={onDismiss}>
+    {alert.message}
+  </Alert>
+);
+
+AlertMessage.propTypes = {
+  alert: PropTypes.shape({
+    isOpen: PropTypes.bool,
```
Comment (rafaelklaessen) [inline src/components/AlertMessage/index.js:14]: Isn't the ` is` redundant? Just `open` would be clear too

File: src/components/InfoPanel/index.js [hunk 2]
```diff
@@ -1,26 +1,35 @@
...
+  subtitleClass: PropTypes.string,
   description: PropTypes.string.isRequired,
 };
 
+InfoPanel.defaultProps = {
+  subtitle: '',
+  subtitleClass: '',
+};
+
+
```
Comment (rafaelklaessen) [inline src/components/InfoPanel/index.js:34]: Unnecessary newline

File: src/scenes/TradeHub/index.js [hunk 3]
```diff
@@ -23,25 +24,39 @@ export default class TradeHub extends Component {
...
     cryptoList: [],
     selectedCrypto: { symbol: 'ETHBTC', baseAsset: 'ETH', quoteAsset: 'BTC' },
+    quantity: 0,
     currentPrice: 0,
     boughtPrice: 0,
     diffPercentage: 0.01,
     highestPrice: 0,
     sellEnabled: false,
     sold: false,
+    alert: { isOpen: true, message: 'Welkom bij de tradingbot', color: 'success' },
```
Comment (rafaelklaessen) [inline src/scenes/TradeHub/index.js:34]: Don't use dutch

File: src/scenes/TradeHub/index.js [hunk 4]
```diff
@@ -23,25 +24,39 @@ export default class TradeHub extends Component {
...
 
+  onDismiss = () => {
+    const alert = Object.assign({}, this.state.alert);
+    alert.isOpen = false;
+    this.setState({ alert });
+  }
+
+  getPercentageChange = (oldNumber, newNumber) => {
+    const decreaseValue = oldNumber - newNumber;
+    return (decreaseValue / oldNumber) * 100;
```
Comment (rafaelklaessen) [inline src/scenes/TradeHub/index.js:57]: The parentheses are redundant

File: src/scenes/TradeHub/index.js [hunk 5]
```diff
@@ -23,25 +24,39 @@ export default class TradeHub extends Component {
...
+  }
```
Comment (rafaelklaessen) [inline src/scenes/TradeHub/index.js:58]: Semicolon

File: src/scenes/TradeHub/index.js [hunk 6]
```diff
@@ -23,25 +24,39 @@ export default class TradeHub extends Component {
...
 
     this.getCryptoList();
     this.bindSocket(this.state.selectedCrypto.symbol);
   };
 
+  onDismiss = () => {
+    const alert = Object.assign({}, this.state.alert);
+    alert.isOpen = false;
+    this.setState({ alert });
+  }
```
Comment (rafaelklaessen) [inline src/scenes/TradeHub/index.js:53]: Semicolon

File: src/scenes/TradeHub/index.js [hunk 7]
```diff
@@ -134,12 +151,16 @@ export default class TradeHub extends Component {
   };
 
   render = () => {
-    const { sellEnabled, selectedCrypto, cryptoList, boughtPrice } = this.state;
+    const {
+      sellEnabled, selectedCrypto, cryptoList, boughtPrice, quantity, currentPrice, highestPrice,
+    } = this.state;
```
Comment (rafaelklaessen) [inline src/scenes/TradeHub/index.js:156]:
```javascript
const {
  sellEnabled,
  selectedCrypto,
  cryptoList,
  boughtPrice,
  quantity,
  currentPrice,
  highestPrice,
} = this.state;
```

**Pull request merged**